app/assets/javascripts/checkbox_control.es6

```diff
@@ -35,6 +38,20 @@ $(function() {
           checkbox.prop("checked", checkboxMemory[index])
         }
       })
+
+      hideableFields.each(function (index) {
+        const hideableField = $(this)
+        if (controlChecked) {
+          if (!hideableField.hasClass('govuk-checkboxes__conditional--hidden')) {
+            hideableField.addClass('govuk-checkboxes__conditional--hidden')
+            hiddenFieldMemory[index] = true
+          }
+        } else {
+          if (hiddenFieldMemory[index]) {
```

Comment:
you can write `} else if (hiddenFieldMemory[index]) {` instead of these 2 lines.